src/main/java/com/redhat/devtools/intellij/commonuitest/fixtures/mainidewindow/toolwindowspane/ProjectExplorer.java

```diff
@@ -48,16 +48,9 @@ public ProjectExplorer(@NotNull RemoteRobot remoteRobot, @NotNull RemoteComponen
      * @return true if the given file exists on the given path in the project
      */
     public boolean isItemPresent(String... path) {
-        try {
-            projectViewTree().expand(path);
-            projectViewTree().clickPath(path, true);
-        } catch (Exception e) {
-            if (!(e instanceof JTreeFixture.PathNotFoundException)) {
-                throw e;
-            }
-            return false;
-        }
-        return true;
+        projectViewTree().expand(path);
+        path[0] = projectViewTree().getValueAtRow(0);
```

Comment:
What is this hack good for?

Reply:
@odockal It is not a hack. Remote-robot team has changed the way how JTreeFixture reads the node values. Instead of using only "project_name_xy" as a root item, you need to have there "project_name_xy  **/path/to/the/project**" as a first item in the path. This can be get from the from the row 0. It is described here - https://github.com/JetBrains/intellij-ui-test-robot/issues/194

Reply:
Maybe I can add some check that the first provided item from the method parameter is really a prefix for the value from row 0, then replace. Otherwise leave it as it is, it will cause the PathNotFoundException. WDYT?

Reply:
Expand can check path existence. So ok, explained. thanks